routes/v1/gardens/gardens_router.go

```diff
@@ -0,0 +1,33 @@
+package gardens_router
+
+import (
+	"fmt"
+	"net/http"
+	"strconv"
+
+	"github.com/gorilla/mux"
+	gardens_controller "github.com/ice-cream-backend/controller/v1/gardens"
+)
+
+func GardensIdGet(w http.ResponseWriter, r *http.Request) {
+	vars := mux.Vars(r)
```

Comment:
@yukari-n-erb Something you can add tomorrow after I push my changes and you merge it but you'll need to add a CORS check when the frontend actually reaches your endpoint. I made a func in the utils folder in my latest PR so you can use that to allow the frontend to reach your endpoints.